public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -107,6 +107,16 @@ export default class FindingDetailsFlyout extends Component<
           this.setState({ correlatedFindings });
         }
       });
+  };
+
+  componentDidMount(): void {
+    this.getIndexPatternId().then((patternId) => {
```

Comment:
Clarification question. Would the use of `.then()` here cause a slight delay when attempting to load the flyout? Would it be better to make use of a `loading` state to generate an "empty" flyout which then gets populated with data once the API call completes?

Reply:
Yeah, makes sense. I have created an issue as this is something that needs a bit more work https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/559